### DISCUSSION TITLE
Android automotive support

### DISCUSSION THREAD

**Discussion by AGo444** [advplyr/audiobookshelf#1638] [Ideas]
New models of Ford, GM, Volvo, Renault, Rivian, and Lucid Motors are running Android Automotive. 
There are no real competitors on Audiobooks in that ecosystem, it might be a great idea if audiobookshelf could be ported to that system. 
What do you think?

**Comment by advplyr** [comment]
I was going to say there is already a feature request for this https://github.com/advplyr/audiobookshelf-app/issues/621

Then I noticed you opened it.

I don't have Android Automotive so it hasn't been something on my list but I'm all for it.

**Reply by dakotadorn12** [reply]
As a new user with a new GM car this would be amazing if possible.

**Comment by slayerBarosh** [comment]
I am already using it via android auto (Germany). I assume Android Automotive is the international name for it?

**Reply by Vito0912** [reply]
No, Android Motive actually runs on the cars hardware. Android Auto runs on your mobile device

**Reply by slayerBarosh** [reply]
Oh, interesting. I didn't even know that existed.

**Reply by rostam420** [reply]
> I am already using it via android auto (Germany). I assume Android Automotive is the international name for it?

I agree I have been using audiobookshelf on andoird auto for around a year (my server is young still) I did find a couple times that I had to add the app to the android auto homescreen via
Settings
Android Auto
Customize Launcher
if you done see audiobookshelf in there just reboot phone or reinstall audiobookshelf
once added should remain until phone gets a wipe

**Comment by Killercollins08** [comment]
I just got a car that has android automotive. Hoping the app becomes a thing so that I can use it natively on me EV. I'd LOVE that!

**Comment by PrzemekSkw** [comment]
Hello, any news about client for automotive? 
Regards.
